deploy/chart/ekuiper/README.md

```diff
@@ -0,0 +1,56 @@
+# eKuiper
+A lightweight IoT edge analytic software
+
+![Version: 1.3.0](https://img.shields.io/badge/Version-1.3.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.5.0](https://img.shields.io/badge/AppVersion-1.5.0-informational?style=flat-square)
+
+## Install the Chart
+
+- From Github
+```
+git clone https://github.com/lf-edge/ekuiper.git
+cd deploy/chart/ekuiper
+helm install my-ekuiper .
+```
+
+- From Helm repo
+```
+helm repo add emqx https://repos.emqx.io/charts
+helm install my-ekuiper emqx/kuiper
```

Comment:
Do we need update repo name to `emqx/ekuiper` ? @ngjaying

Reply:
yes

Reply:
The docker image is lfedge/ekuiper, shall we rename to match that